org.rajnisht7.PopSearch.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "org.rajnisht7.PopSearch",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/search_bar.sh",
+    "finish-args": [
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "PopSearch",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 src/search-bar.py /app/search-bar.py",
+                "install -Dm755 src/search_bar.sh /app/search_bar.sh",
+                "install -Dm644 src/data/PopSearch.desktop /app/share/applications/org.rajnisht7.PopSearch.desktop",
+                "install -Dm644 src/data/org.rajnisht7.PopSearch.png /app/share/icons/hicolor/256x256/apps/org.rajnisht7.PopSearch.png",
+                "install -Dm644 -t /app/dependencies src/dependencies/*.whl"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rajnisht7/PopSearch-Release/archive/refs/tags/v0.1.tar.gz",
+                    "sha256": "6067af5171b942f076aee40eeb8632a4f0473c777f70119206cb7c617dc4db33",
+                    "x-checker-data": {
+                        "type": "github",
+                        "user": "rajnisht7",
+                        "repo": "PopSearch-Release",
+                        "tag-pattern": ".*"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "python-dependencies",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --prefix=/app --no-index --find-links=/app/dependencies certifi charset_normalizer idna requests urllib3"
+            ],
+            "sources": [
+                {
+                    "type": "files",
+                    "path": "/app/dependencies"
```

Comment:
1. this path is incorrect.
2. type `files` doesn't exists. Not sure which part of the documentation is wrong to tell you this is valid.
3. why is there a bunch of python wheels checked in? They don't belind here.

Reply:
Point 3 was requested in https://github.com/flathub/flathub/pull/6200#issuecomment-2676234190 already